freetextresponse/tests.py

```
@@ -260,6 +261,23 @@ def test_compute_score(self, credit):
             {'value': credit.value, 'max_value': Credit.full.value},
         )
 
+    @ddt.data(Credit.zero, Credit.half, Credit.full)
+    def test_compute_score_error(self, credit):
+        # pylint: disable=protected-access
+        """
+        Tests _compute_score's error handling
```

Same here too, let's use IntegrityError in the first line of the comment.  And be specific in the other parts of the comment as well. 

Does this test actually test that all exceptions are handle?  Or is it just the one IntegrityError?

---

The current test only checks for the IntegrityError exception. The try - except that I introduced also only handles the IntegrityError exception. I could have it catch and ignore all exceptions, but I don't know if that is a good idea.

---

@jlikhuva Yeah, I think that's what @caseylitton is pointing out; the comment implies that this handles more than it actually does. We should only be catching that specific exception in this case; let's update the docs to reflect what we're catching.

---

Yes, thanks stv.  

@jlikhuva Specify exactly what's happening in the function.  We're forcing runtime.publish to throw an IntegrityError in order to test that compute_score passes it without issue.